tests/build-newsroom-videos.test.js

```diff
@@ -1,22 +1,23 @@
-const { readFileSync, rmSync, mkdirSync } = require('fs');
-const { resolve } = require('path');
+const { readFileSync, removeSync, mkdirpSync } = require('fs-extra');
+const { resolve, join } = require('path');
 const { buildNewsroomVideos } = require('../scripts/build-newsroom-videos');
 const { mockApiResponse, expectedResult } = require('./fixtures/newsroomData');
 const fetch = require('node-fetch-2');
+const os = require('os');
 
 jest.mock('node-fetch-2', () => jest.fn());
 
 describe('buildNewsroomVideos', () => {
-    const testDir = resolve(__dirname, 'test_config');
+    const testDir = join(os.tmpdir(), 'test_config');
```

Comment:
This is a nice addition. We should use `os.tmpDir()` for other scripts as well where a temp folder was being created inside the website directory.